app/admin/allocation_chart.rb

```diff
@@ -11,9 +13,36 @@
           column(I18n.t('name'), :user)
           column(I18n.t('skills')) { |allocation| allocation.user.skills.map(&:title).join(', ') }
           column(I18n.t('client'), :project)
-          column(I18n.t('allocated_until'), :end_at)
+          column(I18n.t('allocated_until'), class: 'allocated-column') do |allocation|
+            build_allocation_status_cell(allocation)
+          end
         end
       end
     end
   end
 end
+
+def build_allocation_status_cell(last_allocation)
+  allocation_status = ActiveAdmin::AllocationChartHelper.allocation_status_for(last_allocation: last_allocation)
+
+  column_cell_base(
+    content: allocation_status_content(last_allocation: last_allocation,
+                                       allocation_status: allocation_status), allocation_status: allocation_status
+  )
+end
+
+def allocation_status_content(last_allocation:, allocation_status:)
+  return last_allocation.end_at if last_allocation.end_at
+
+  if allocation_status == ActiveAdmin::AllocationChartHelper::Status::NOT_ALLOCATED
```

Comment:
I think we can make this if/else statement a bit cleaner!

How about passing this check into a smaller private method and turning this into a ternary check? like so:
```ruby
if unallocated?(allocation_status) ? I18n.t('not_allocated') : I18n.t('allocated')
...
private def unallocated?(status)
  status == ActiveAdmin::AllocationChartHelper::Status::NOT_ALLOCATED
end
```
That way we improve the readabilty of this `if` statement and we reduce it to a single line